packages/react-devtools-extensions/src/backend.js

```diff
@@ -7,6 +7,10 @@
 'use strict';
 
 let welcomeHasInitialized = false;
+const _window = {
+  addEventListener: window.addEventListener.bind(window),
```

Comment:
The original logic in the Highlighter checks whether `window` and `window.addEventListener` exist first. Do we need to do the same here? I guess it's fine to assume they always exist, because this file should only run on browser (extension).